### PR TITLE
Do static setup for geo demo

### PR DESCRIPTION
**Pull request by wolffcm**
- Created a skeleton for the app
- Create a list of customers
- Randomly generate some polygons

### CODE REVIEW

File: examples/geospatial/client/geospatial/BidGenerator.java [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+        AdBrokerBenchmark.connect(m_client, config.servers);
+
+        m_bidRegions = getRandomRegions();
+    }
+
+    /**
+     * For now, just produce rectangular regions.
+     */
+    private List<GeographyValue> getRandomRegions() {
+        Random rand = new Random(777);
```
Comment (BillWhite) [inline examples/geospatial/client/geospatial/BidGenerator.java]: Should this have a geography point as a parameter?  The bids are going to be localized around the store's location, aren't they?  Maybe that's more realistic than you wanted.

File: examples/geospatial/advertisers.csv [hunk 2]
```diff
@@ -0,0 +1,500 @@
...
+490, "Aadhya's Wonderful Jewelry"
+491, "AanyaAarav's Superior Clothes"
+492, "Abigail's Extraordinary Electronics"
+493, "Aditya's Stupendous Sporting Goods"
+494, "Ahmed's Excellent Foods"
+495, "Ali's Unmatched Jewelry"
+496, "Amelia's Magnificient Clothes"
+497, "Aradhya's Sterling Electronics"
+498, "Arjun's Distinguished Sporting Goods"
+499, "Brigit's Fine Foods"
```
Comment (BillWhite) [inline examples/geospatial/advertisers.csv]: Nice names, but I think you need ukulele stores.

**Comment by BillWhite** [comment]
That's all I have.  It looks like you are on your way.

File: examples/geospatial/client/geospatial/BidGenerator.java (hunk 1 above)
Comment (wolffcm) [inline examples/geospatial/client/geospatial/BidGenerator.java]: Sorry for the delay in replying to your comment, I got distracted by other stuff. 

I was thinking that each bid would just choose randomly from some number of randomly generated polygons.  In practice you could imagine showing an ad when a user is near the advertising store or the store of a competitor.  But I'm not so concerned about realism at this point.

**Comment by wolffcm** [comment]
I'll be rebasing and squashing this work before merging to the topic branch.

**Pull request closed without merging**